README.md

```diff
@@ -28,3 +28,46 @@ export default {
 ```
 
 Check the examples/minimal folder for a complete example.
+
+## Gotchas and/or Limitations
+
+### Provide unique names for Modules and Tests
+
+Names (and assertion messages) are used as unique identifiers to build up the test results that @web/test-runner is expecting. Most of the actions to build this data structure are only appending new data. Take for example the following test code:
+
+```
+QUnit.module('Module A', () => {
+  QUnit.test('test 1', assert => {
+    assert.true(true, 'assertion 1')
+  })
+  QUnit.test('test 2', assert => {
+    assert.true(true, 'assertion 2')
+  })
+})
+QUnit.module('Module A', () => {
+  QUnit.test('test 1', assert => {
+    assert.true(true, 'assertion 3')
+  })
+})
+```
+
+This would result in the following summary output:
+
+```
+ Module A [Chrome]
+  test 1 [Chrome]
+    ✓ assertion 1
+    ✓ assertion 3
+  test 2 [Chrome]
+    ✓ assertion 2
+```
+
+### Progress Reporting
+
+Progress isn't reported per test but instead is all at once. This currently appears to be a limitation of this type of custom runner for @web/test-runner.
+
```

Comment:
This sentence is imho a little bit confusing. WTR prints progress at least per test file.

Reply:
Good point. I'll update it to indicate it is at best per test file.